vllm/outputs.py

```diff
@@ -223,7 +223,11 @@ def from_seq_group(
             if delta:
                 # Slice logprobs delta if applicable
                 if output_logprobs:
-                    output_logprobs = output_logprobs[-num_output_tokens:]
+                    # Can happen when n > 0 and request finishes before the othere
```

Comment:
```suggestion
                    # Can happen when n > 0 and request finishes before the others
```
the line is also too long (failing pre-commit)